### DISCUSSION TITLE
GraphQL API should have a isDraft field for Pull Requests

### DISCUSSION THREAD

**Discussion by brunolemos** [community/community#24816] [Apps, API and Webhooks]
<p>It’s currently not possible to tell if the pull request is a draft or not via api v4.</p>
<p>API v3 returns a <em>draft</em> field.</p>

**Comment by AndreaGriffiths11** [comment]
<p>Thanks for the feedback! I’ve taken your suggestion and passed it along to the appropriate teams. Thanks again for reaching out :grinning:</p>

**Comment by brunolemos** [comment]
<p>Could you bump this suggestion please? It’s such a simple one, and annoying to have to make new api requests to the old api to get this info.</p>

**Comment by brunolemos** [comment, marked as answer]
<p>This is now available under Preview by passing the <strong>application/vnd.github.shadow-cat-preview+json</strong>  Accept header: <a href="https://developer.github.com/v4/previews/#draft-pull-requests-preview" rel="nofollow noopener">https://developer.github.com/v4/previews/#draft-pull-requests-preview</a></p>
<p>Example query:</p>
<pre><code>{
  pr: resource(url: "https://github.com/react-spring/react-spring/pull/808") {
    ... on PullRequest {
      title
      isDraft
    }
  }
}
</code></pre>
<p>Response:</p>
<pre><code>{
  "data": {
    "pr": {
      "title": "feat: add SpringValue class",
      "isDraft": true
    }
  }
}
</code></pre>
